client/ayon_blender/api/lib.py

```diff
@@ -311,23 +311,44 @@ def get_selected_collections():
     return [id for id in ids if isinstance(id, bpy.types.Collection)]
 
 
-def get_selection(include_collections: bool = False) -> List[bpy.types.Object]:
+def get_selection(
+        include_collections: bool = False,
+        include_object_children_recursive: bool = True
```

Comment:
Please default this to `False` - This is an odd default for a library function that is set out to return your current selection. It makes no sense from the perspective of the library function. If you want it enabled somewhere, then pass the argument and set it to True.